modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayTeamAdministrator.testcase

```diff
@@ -165,10 +165,10 @@ definition {
 				numberItems = "20",
 				valuePagination = "150");
 
-			TestrayManage.specificVersion(
-				caseName = "",
+			TestrayManage.createTeam(
 				manageOption = "Manage Teams",
-				nameOptions = "Projects_1 Project");
+				nameOptions = "Projects_1 Project",
+				version = "");
```

Comment:
Hi @joelrocha68, on this macro, you don't need to use an empty variable. What do you think of use the `isSet` function?